testing/infra/terraform/modules/ec_deployment/variables.tf

```diff
@@ -1,7 +1,7 @@
 # Deployment settings
 
 variable "deployment_name_prefix" {
-  default     = "apmserver"
+  default     = "apmserver-benchmarks"
```

Comment:
required? It seems the folder structure is agnostic about the benchmarks

Reply:
reverted